e2e/tests/functional/recentObjects.e2e.spec.js

```diff
@@ -102,31 +101,117 @@ test.describe('Recent Objects', () => {
             hasText: myItemsFolderName
         }).click();
 
         // Verify that the hash URL updates correctly
         await waitForMyItemsNavigation;
-        // eslint-disable-next-line no-useless-escape
-        expect(page.url()).toMatch(new RegExp(`.*mine\?.*`));
+        expect(page.url()).toMatch(new RegExp(`.*mine?.*`));
     });
-    test.fixme("Clicking on the 'target button' scrolls the object into view in the tree and highlights it", async ({ page }) => {
+    test("Clicking on the 'target button' scrolls the object into view in the tree and highlights it", async ({ page }) => {
+        const clockTreeItem = page.getByRole('tree', { name: 'Main Tree'}).getByRole('treeitem', { name: clock.name });
+        const folderTreeItem = page.getByRole('tree', { name: 'Main Tree'})
+            .getByRole('treeitem', {
+                name: folderA.name,
+                expanded: true
+            });
+
+        // Click the "Target" button for the Clock which is nested in a folder
+        await page.getByRole('button', { name: `Open and scroll to ${clock.name}`}).click();
+
+        // Assert that the Clock parent folder has expanded and the Clock is visible)
+        await expect(folderTreeItem.locator('.c-disclosure-triangle')).toHaveClass(/--expanded/);
+        await expect(clockTreeItem).toBeVisible();
+
+        // Assert that the Clock treeitem is highlighted
+        await expect(clockTreeItem.locator('.c-tree__item')).toHaveClass(/is-targeted-item/);
+
+        // Wait for highlight animation to end
+        await waitForAnimations(clockTreeItem.locator('.c-tree__item'));
+
+        // Assert that the Clock treeitem is no longer highlighted
+        await expect(clockTreeItem.locator('.c-tree__item')).not.toHaveClass(/is-targeted-item/);
     });
-    test.fixme("Tests for context menu actions from recent objects", async ({ page }) => {
+    test("Persists on refresh", async ({ page }) => {
+        assertInitialRecentObjectsListState();
+        await page.reload();
+        assertInitialRecentObjectsListState();
     });
+    test("Displays objects and aliases uniquely", async ({ page }) => {
+        const mainTree = page.getByRole('tree', { name: 'Main Tree'});
+
+        // Navigate to the clock and reveal it in the tree
+        await page.goto(clock.url);
+        await page.getByTitle('Show selected item in tree').click();
+
+        // Right click the clock and create an alias using the "link" context menu action
+        const clockTreeItem = page.getByRole('tree', {
+            name: 'Main Tree'
+        }).getByRole('treeitem', {
+            name: clock.name
+        });
+        await clockTreeItem.click({
+            button: 'right'
+        });
+        await page.getByRole('menuitem', {
+            name: /Create Link/
+        }).click();
+        await page.getByRole('tree', { name: 'Create Modal Tree'}).getByRole('treeitem').first().click();
+        await page.getByRole('button', {
```

Comment:
i think we should consider expanding the number of lines per row to allow this locator to be a one liner

Reply:
We can compress this to a single line, that was me manually adding endlines.

Top of build 8 we should get prettier implemented. I have it mostly done on a branch